python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -108,22 +108,103 @@ def _apply_nms(orig_prediction: dict, iou_thresh: float = 0.5):
+    two objects:
+
+    [{'box': {'topX': 0.645,
+              'topY': 0.373,
+              'bottomX': 0.8276,
+              'bottomY': 0.6102
+                },
+      'label': 'can',
+      'score': 0.945},
+     {'box': {'topX': 0.324,
+              'topY': 0.5643,
+              'bottomX': 0.6511,
+              'bottomY': 0.865
+                },
+      'label': 'milk_bottle',
+      'score': 0.93}
+    ]
+    The bbox coordinates need to be scaled by the image dimensions.
+    """
+
+    x, y = image_size
+
+    boxes = []
+    scores = []
+    labels = []
+    for detection in image_detections:
+        # predicted class label
+        label = detection['label']
+        # confidence score from model
+        score = detection['score']
+        # predicted bbox coordinates
+        box = detection["box"]
+
+        ymin, xmin, ymax, xmax = (
+            box["topY"], box["topX"], box["bottomY"], box["bottomX"])
+
+        # the automl mlflow model generates normalized bbox coordinates
+        # they need to be scaled by the size of the image
+        x_min_scaled, y_min_scaled = x * xmin, y * ymin
+        x_max_scaled, y_max_scaled = x * xmax, y * ymax
+
+        scores.append(score)
+        labels.append(label)
+        boxes.append([x_min_scaled, y_min_scaled, x_max_scaled, y_max_scaled])
+    try:
+        labels = [int(x) for x in labels]
+    except BaseException:
+        labels = [label_dict[x] for x in labels]
+
+    return {
+        "boxes": Tensor(boxes),
+        "labels": Tensor(labels),
+        "scores": Tensor(scores)}
+
+
+def _wrap_image_model(model, examples, model_task, is_function,
+                      number_of_classes: int = None,
+                      classes: Union[list, np.array] = None):
     """If needed, wraps the model or function in a common API.
 
     Wraps the model based on model task and prediction function contract.
 
     :param model: The model or function to evaluate on the examples.
     :type model: function or model to wrap
     :param examples: The model evaluation examples.
-    :type examples: ml_wrappers.DatasetWrapper or numpy.ndarray
-        or pandas.DataFrame or panads.Series or scipy.sparse.csr_matrix
-        or shap.DenseData or torch.Tensor
+    :type examples: ml_wrappers.DatasetWrapper
+                    or numpy.ndarray
+                    or pandas.DataFrame or panads.Series
+                    or scipy.sparse.csr_matrix
+                    or shap.DenseData
+                    or torch.Tensor
     :param model_task: Parameter to specify whether the model is an
-        'image_classification' or another type of image model.
+                       image_classification' or another type of image model.
     :type model_task: str
-    :param number_of_classes: optional parameter specifying the number of classes in
-        the dataset
+    :param classes: optional parameter specifying a list of class names
+                    the dataset
```

Comment:
nit: the doc here seems a bit messed up, it looks like the in word is missing and there are some whitespace issues